services/web/src/screens/Onboard/index.js

```diff
@@ -77,49 +77,52 @@ export default function OnboardScreen() {
     }
   }
 
-  function render() {
-    return (
-      <React.Fragment>
-        <Meta title="Tell Us More" />
-        <LogoTitle title="Tell Us More" />
-        <Form loading={loading} onSubmit={onSubmit} noValidate>
-          <Segment.Group>
-            <Segment padded>
-              {error?.type !== 'validation' && <ErrorMessage error={error} />}
-              {!user.email && (
-                <EmailField
-                  name="email"
-                  value={body.email || ''}
-                  onChange={setField}
-                  error={error}
-                />
-              )}
-              {!user.phone && (
-                <PhoneField
-                  name="phone"
-                  value={body.phone || ''}
-                  onChange={setField}
-                  error={error}
-                />
-              )}
-              <Form.Button
-                fluid
-                primary
-                size="large"
-                content="Continue"
-                loading={loading}
-                disabled={loading}
-              />
-            </Segment>
-          </Segment.Group>
-        </Form>
-      </React.Fragment>
-    );
-  }
-
   if (isValidUser(user)) {
     return <Redirect to="/" />;
   }
 
-  return render();
+  return (
+    <Stack>
+      <Meta title="Tell Us More" />
+      <Logo />
+      <form
```

Comment:
I don't get it we have components for `Anchor` and `Text` but we don't need one for `Form`? :)

Reply:
the form element does not have styles in mantine world